qt/scientific_interfaces/Inelastic/Processor/ElwinPresenter.cpp

```diff
@@ -218,6 +217,7 @@ void ElwinPresenter::handleRun() {
 
   // Set the result workspace for Python script export
   m_pythonExportWsName = m_outputNamePresenter->generateOutputLabel() + "_elwin_eq2";
+  AnalysisDataService::Instance().remove("specWSnext");
```

Comment:
The `specWSnext` workspace can be seen in the ADS after running the Elwin tab. I believe this line needs to go into the `ElwinPresenter::runComplete` method because it is the callback for when processing in the background thread has completed